ingest/indexer/service/blockprocessor/export_test.go

```diff
@@ -0,0 +1,42 @@
+package blockprocessor
+
+import (
+	"github.com/cosmos/cosmos-sdk/types"
+
+	commondomain "github.com/osmosis-labs/osmosis/v26/ingest/common/domain"
+	"github.com/osmosis-labs/osmosis/v26/ingest/indexer/domain"
+)
+
+func NewBlockUpdatesIndexerBlockProcessStrategy(blockUpdateProcessUtils commondomain.BlockUpdateProcessUtilsI, client domain.Publisher, poolExtractor commondomain.PoolExtractor, poolPairPublisher domain.PairPublisher) *blockUpdatesIndexerBlockProcessStrategy {
+	return &blockUpdatesIndexerBlockProcessStrategy{
+		blockUpdateProcessUtils: blockUpdateProcessUtils,
+		client:                  client,
+		poolExtractor:           poolExtractor,
+		poolPairPublisher:       poolPairPublisher,
+	}
+}
+
+type BlockUpdatesIndexerBlockProcessStrategy = blockUpdatesIndexerBlockProcessStrategy
```

Comment:
I was not able to find usages af new alias type, should we make `blockUpdatesIndexerBlockProcessStrategy` exported and return it as exported instead of creating new alias type?

Reply:
Hi @deividaspetraitis , as you may know, this export_test.go is to expose/export those private functions for test purposes. They are currently used here:

https://github.com/osmosis-labs/osmosis/blob/1a1ac536da82c042a4de0c23070a50ad0d8deac8/ingest/indexer/service/blockprocessor/full_indexer_block_process_strategy_test.go#L227

https://github.com/osmosis-labs/osmosis/blob/1a1ac536da82c042a4de0c23070a50ad0d8deac8/ingest/indexer/service/blockprocessor/block_updates_indexer_block_process_strategy_test.go#L139